tools/buildsteps/rbpi/configure-xbmc

```diff
@@ -2,4 +2,4 @@ WORKSPACE=${WORKSPACE:-$( cd $(dirname $0)/../../.. ; pwd -P )}
 XBMC_PLATFORM_DIR=rbpi
 . $WORKSPACE/tools/buildsteps/defaultenv
 
-make -C $WORKSPACE/tools/depends/target/cmakebuildsys
+platform="-DCORE_PLATFORM_NAME=rbpi" make -C $WORKSPACE/tools/depends/target/cmakebuildsys
```

Comment:
In my opinion passing `CORE_PLATFORM_NAME` to kodi configure has to be handled automatically by depends